### DISCUSSION TITLE
Principais eventos de tecnologia

### DISCUSSION THREAD

**Discussion by vNNi** [frontendbr/forum#1347] [Discussão]
Na opinião de vocês quais são os melhores eventos de tecnologia nacionais e internacionais?
Levando em conta: 

- Conteúdo das palestras (se tiverem); 

- Estrutura;

- Local;

**Comment by rsmelo92** [comment]
A Jusbrasil tem um repositório de eventos, tem alguns interessantes já mas estão sendo adicionados mais aos poucos _(quem quiser pode contribuir tb)_
https://github.com/jusbrasil/tech-calendar

**Comment by ViniciusGularte** [comment]
O melhor que tive oportunidade de ir até agora foi o **BrazilJs,** não sei como tava ano passado, mas no ano que fui em POA (2017) a organização tava show, e palestras ótimas. É simplesmente o maior evento de JavaScript do mundo hehe, esse evento me fez ver como tava na zona de conforto na época e correr atrás de conhecimento.

- Conteúdo das palestras (se tiverem): Javascript :heart: 
- Estrutura: Teve espaço pra todo mundo, deu um pouco de confusão na hora do coffe break, mas é normal pela quantidade de gente.
- Local: Em POA eles fazem la no Barra Shopping Sul, lembro que a fila para entrar no primeiro dia dava voltas .
Pretendo juntar dinheiro pra ir denovo esse ano, e no **fisl** também, não sei se tem grupo aqui pra juntar o pessoal pra fazer caravana nesses eventos(o que ia ser bem legal)

**Comment by rodineicosta** [comment]
@rsmelo92, enviei um PR no repo do JusBrasil com informações do **FLISOL Castanhal 2019**, que ocorrerá em abril no estado do Pará.

**Comment by isacmoura** [comment]
Ainda não fui em muitos, mas o melhor para mim até agora foi a Python Brasil 2018. Esse ano vou pra CPBR 12. Aqui tem um calendário de eventos: https://github.com/dunossauro/calendario-de-eventos quem quiser adicionar mais alguns é só mandar o PR

**Comment by erickmajor** [comment]
Poderíamos aproveitar o comentário do @vNNi e criar um calendário unificado (Google Calendar, p.ex.) de eventos da área de programação.

O que acham?

**Comment by isacmoura** [comment]
@erickmajor acho uma boa. Bem mais prático, ao meu ver. Podemos unir as informações dos repositórios desta issue num só Calendar.

**Comment by felipefialho** [comment]
Dentro do @frontendbr temos uma página de [eventos com a temática Front-end](https://eventos.frontendbr.com.br)

**Comment by erickmajor** [comment]
Fala Galera, fiz uma prévia: http://bit.do/eGG3q

Espero que curtam! Aceito colaboradores para adicionar e/ou editar os eventos tbm.
